src/main/java/bio/terra/cli/command/resources/Create.java

```diff
@@ -0,0 +1,43 @@
+package bio.terra.cli.command.resources;
+
+import bio.terra.cli.auth.AuthenticationManager;
+import bio.terra.cli.context.CloudResource;
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+import bio.terra.cli.service.WorkspaceManager;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the third-level "terra resources create" command. */
+@Command(name = "create", description = "Create a new controlled resource.")
+public class Create implements Callable<Integer> {
+
+  @CommandLine.Option(
+      names = "--type",
```

Comment:
should all option params have `--` prefixes? I didn't do that for notebooks.

Reply:
I was following gcloud example (e.g. --project, --help, --billing-project). I don't have a strong opinion either way, but we should be consistent across commands.